docs/index.md

```diff
@@ -1,241 +1,172 @@
 
-Logfire also has an out-of-the-box Pydantic integration that lets you understand the data passing through your Pydantic models and get analytics on validations.
+## Tracing with Spans
 
-We can record Pydantic models directly:
+Spans let you add context to your logs and measure code execution time. Multiple spans combine to form a trace,
+providing a complete picture of an operation's journey through your system.
 
 ```py
-from datetime import date
+from pathlib import Path
 import logfire
-from pydantic import BaseModel
+
+cwd = Path.cwd()
+total_size = 0
 
 logfire.configure()
 
-class User(BaseModel):
-    name: str
-    country_code: str
-    dob: date
+with logfire.span('counting size of {cwd=}', cwd=cwd):
+    for path in cwd.iterdir():
+        if path.is_file():
+            with logfire.span('reading {file}', file=path):
+                total_size += len(path.read_bytes())
 
-user = User(name='Anne', country_code='USA', dob='2000-01-01')
-logfire.info('user processed: {user!r}', user=user)  # (1)!
+    logfire.info('total size of {cwd} is {size} bytes', cwd=cwd, size=total_size)
 ```
 
-1. This will show `user processed: User(name='Anne', country_code='US', dob=datetime.date(2000, 1, 1))`, but also allow you to see a "pretty" view of the model within the Logfire Platform.
-
-![Logfire pydantic manual screenshot](images/index/logfire-screenshot-pydantic-manual.png)
-
-Or we can record information about validations automatically:
+In this example:
 
-```py
-from datetime import date
-import logfire
-from pydantic import BaseModel
+1. The outer span measures the time to count the total size of files in the current directory (`cwd`).
+2. Inner spans measure the time to read each individual file.
+3. Finally, the total size is logged.
 
-logfire.configure(pydantic_plugin=logfire.PydanticPlugin(record='all'))  # (1)!
+![Counting size of loaded files screenshot](images/logfire-screenshot-first-steps-load-files.png)
 
-class User(BaseModel):
-    name: str
-    country_code: str
-    dob: date
-
-User(name='Anne', country_code='USA', dob='2000-01-01')  # (2)!
-User(name='Ben', country_code='USA', dob='2000-02-02')
-User(name='Charlie', country_code='GBR', dob='1990-03-03')
-```
+By instrumenting your code with traces and spans, you can see how long operations take, identify bottlenecks,
+and get a high-level view of request flows in your system — all invaluable for maintaining the performance and
+reliability of your applications.
 
-1. This configuration means details about all Pydantic model validations will be recorded. You can also record details about validation failures only, or just metrics; see the [pydantic plugin docs][logfire.PydanticPlugin].
-2. Since we've enabled the Pydantic Plugin, all Pydantic validations will be recorded in Logfire.
+[conda]: https://conda.io/projects/conda/en/latest/user-guide/install/index.html
```

Comment:
This ends a bit abruptly, I think we need a "Next Steps" section, where we say:
* the real power of logfire comes from integrations with many popular libraries, learn more
* as well as spans, you can use logfire to record metrics, learn more
* of course Logfire doesn't just work with Python, learn more

Reply:
Done 👍